### PR TITLE
[Feature] Player controller / Level 1 Blockout Map

### PR DESCRIPTION
**Pull request by J2ExSlayer**
I Created a Player Controller (With a bunch of testing scripts that I didn't delete or finish ) and Finished a bare minimum blockout of level 1

### CODE REVIEW

File: Assets/Scripts/ScriptableObjects/TestingControllers/PlayerController01.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerController01 : MonoBehaviour
```
Comment (KadynCBR) [inline Assets/Scripts/ScriptableObjects/TestingControllers/PlayerController01.cs:6]: PlayerController01

not sure which to evaluate, it looks like playercontroller04 is the main one(?)

File: Assets/Scripts/ScriptableObjects/TestingControllers/MouseLook.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class MouseLook : MonoBehaviour
```
Comment (KadynCBR) [inline Assets/Scripts/ScriptableObjects/TestingControllers/MouseLook.cs:6]: This is definitely something that should be built using Cinemachine


https://docs.unity3d.com/Packages/com.unity.cinemachine@2.1/api/Cinemachine.CinemachinePOV.html

File: Assets/Scripts/ScriptableObjects/TestingControllers/PlayerCam.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerCam : MonoBehaviour
```
Comment (KadynCBR) [inline Assets/Scripts/ScriptableObjects/TestingControllers/PlayerCam.cs:5]: Same as mouseLook, no real reason to make this from scratch unless you were doing something incredibly artisanal and still cinemachine handles a lot.

File: Assets/Scripts/ScriptableObjects/TestingControllers/PlayerMovement.cs [hunk 4]
```diff
@@ -0,0 +1,72 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class PlayerMovement : MonoBehaviour
```
Comment (KadynCBR) [inline Assets/Scripts/ScriptableObjects/TestingControllers/PlayerMovement.cs:6]: Not sure If this is supposed to be different? "player movement sounds like its not a testing controller(?)

File: Assets/Scripts/ScriptableObjects/TestingControllers/PlayerMovement.cs [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        playerInputs.Enable();
+    }
+
+
+    private void OnDisable()
+    {
+        playerInputs.Disable();
+    }
+
+    private void OnCollisionEnter(Collision other)
```
Comment (KadynCBR) [inline Assets/Scripts/ScriptableObjects/TestingControllers/PlayerMovement.cs:48]: There seems to quite a few ways of checking the ground, I'm happy you evaluated many different methods but I would request that we consolidate these to one preferred method.

**Review by KadynCBR — APPROVED** [review]
https://docs.unity3d.com/Manual/class-ScriptableObject.html
"ScriptableObjects" has a very specific meaning in unity, making it a folder title where you place scripts might be a bit confusing as these additions in the scriptableobjects folder are not _scriptableobjects_ 


A lot of inconsistent and duplicate scripts, I understand wanting to test and evaluate but in terms of maintainability this quickly becomes a nightmare because 
which playercontroller do i use? playerController01 or playercontroller04, does Playermovement01 differ from playermovement02? 

its okay to commit things to git and remove them. I would ask for you to remove these duplicate files

File: Assets/Scripts/ScriptableObjects/TestingControllers/PlayerController01.cs (hunk 1 above)
Comment (J2ExSlayer) [reply Assets/Scripts/ScriptableObjects/TestingControllers/PlayerController01.cs:6]: Yes playercontroller04 is the main one, or one that we are currently using which is in the character controller folder, the others were tests till I found one that I liked which i put in the testing controllers folder, will probably delete or deal with these in the future, but will definitely be adding more testing scripts eventually, will be looking into ways of managing my testing scripts for the future.

File: Assets/Scripts/ScriptableObjects/TestingControllers/MouseLook.cs (hunk 2 above)
Comment (J2ExSlayer) [reply Assets/Scripts/ScriptableObjects/TestingControllers/MouseLook.cs:6]: I don't know what Cinemachine is will look into it, was just following along with a tutorial, I did not end up using this script, it was one the testing ones.

File: Assets/Scripts/ScriptableObjects/TestingControllers/PlayerCam.cs (hunk 3 above)
Comment (J2ExSlayer) [reply Assets/Scripts/ScriptableObjects/TestingControllers/PlayerCam.cs:5]: Same as previous ^

**Pull request merged**